pegasus/sites.v3/code.org/public/teach.haml

```diff
@@ -104,6 +104,8 @@ theme: responsive_full_width
       %a.link-button.secondary{href: CDO.studio_url("/catalog")}
         =hoc_s(:call_to_action_see_full_curriculum_catalog)
 
+= view :ai_pl_skinny_banner
```

Comment:
Moved this banner down here:

<img width="1088" alt="Screenshot 2023-11-21 at 10 49 53 AM" src="https://github.com/code-dot-org/code-dot-org/assets/9256643/db674e95-af0e-43c6-a973-876a91d5b97d">